tests/renderer/test_renderer_in_context.py

```
@@ -0,0 +1,25 @@
+import pytest
+
+from htmy import Context, component
+from htmy.renderer import BaselineRenderer, Renderer
+from htmy.renderer.context import RendererContext
+from htmy.renderer.typing import RendererType
+
+
+@pytest.mark.asyncio
+@pytest.mark.parametrize(
+    ("renderer",),
```

Ruff (0.14.8)

Wrong type passed to first argument of `pytest.mark.parametrize`; expected `str`

Use a string for the first argument

(PT006)